test/unit/compose/app.spec.ts

```diff
@@ -1294,29 +1293,31 @@ describe('compose/app', () => {
 				.to.deep.include({ serviceName: 'main' });
 		});
 
-		it('should not try to start a container which has exited and has restart policy of no', async () => {
+		it('should not try to start a container which has exited', async () => {
```

Comment:
Maybe I'm misunderstanding this test change, but wouldn't we want to restart a container that has exited with an error code if the restart policy is `always` or `on-failure`? Doesn't this test case go against that?

Reply:
The supervisor never restarts a container that has already been started before. The restart policy is handled by the engine. The only exception is for the engine/hostOS race bug https://github.com/balena-os/balena-supervisor/blob/81b307510d758c9390836acd2d8547e1263eaf35/src/compose/app.ts#L747